src/main/java/co/org/osso/api/DemoController.java

```diff
@@ -0,0 +1,57 @@
+//package co.org.osso.api;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.HashMap;
+import java.util.Map;
+
+// this classes small and delegate to services
```

Comment:
I think you can delete this class now, you've got a much better example in the other controller.

Reply:
DOne